src/shared/output-mode.c

```diff
@@ -1,8 +1,32 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
+#include <stdlib.h>
+
+#include "log.h"
 #include "output-mode.h"
 #include "string-table.h"
 
+void getenv_output_mode(OutputMode *output_mode) {
+        OutputMode mode;
+        const char *e;
+
+        assert(output_mode);
+
+        e = getenv("SYSTEMD_JOURNAL_OUTPUT_MODE");
+        if (!e)
+                return;
+
+        mode = output_mode_from_string(e);
+        if (mode < 0) {
+                log_debug_errno(mode,
+                                "Unknown output format '%s' specified in '$SYSTEMD_JOURNAL_OUTPUT_MODE', ignoring.",
```

Comment:
Add a `SYNTHETIC_ERRNO(EINVAL)` here?

Reply:
Not necessary, as this function does not return error code.